docs/assertions/custom-assertions.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: Custom Assertions
+---
+
+Pester allows users to create their own `Should`-operators for more advanced assertions. This is done by defining a test-function and registering it with Pester using the guidelines below.
+
+## Create the function
+
+### Requirements / Interface
+
+[Should](../commands/Add-ShouldOperator) will automatically pass on values from pipeline to your function as `ActualValue`.
+This can be an array or a single object depending on whether you register the function with `-SupportsArrayInput`.
+
+When `Should` calls your custom assertion, it will invoke it with the following parameters:
+- **ActualValue**: The value passed to `Should` to assert. 
+- **Negate**: Internal switch/bool parameter used to delcare if the user called `Should -Not ...`.
+- **CallerSessionState**: Used by some internal assertions to invoke code in the user's state. This can usually be ignored.
+- *Remaining bound parameters*, typically matching the parmaeters your accept in your assertion, like `ExpectedValue`, `Because`, `MyCustomParameter` etc.
```

Comment:
```suggestion
- *Remaining bound parameters*, typically matching the parameters your accept in your assertion, like `ExpectedValue`, `Because`, `MyCustomParameter` etc.
```